### ISSUE TITLE
Not working at all - BOP also installed

### ISSUE THREAD

**Issue by Outfluence** [superckl/BiomeTweaker#36]
I added a file to reference in the BiomeTweaker.cfg file called testing,cfg
I updated that code to read- "include":[testing.cfg]

The testing.cfg file has example code pasted into it. The code to turn every biome into mostly desert. I can post it if necessary.

I want to use the biomeTweaker features but I can not even make the example work for me. When I create a new world (in any world type, Default or BOP) the world spawns in normally with no changes.

I am running Minecraft Launcher 1.6.13 and release version 1.7.10,with Mods BiomesTweaker-1.7.10-1.1 and BiomesOPlenty-1.7.10-2.1.0.1387-universal, through forge-1.7.10-10.13.2.1291-installer

Any suggestions are appreciated!

**Comment by superckl** [comment]
Your config should read

    "include":["testing.cfg"]

Please add the quotes and get back to me.

**Comment by Outfluence** [comment]
Great catch! However, it seemed to make no change in the world generation...

This is the entire testing.cfg file

#Specify ids
desert = forBiomes(2)
hills = forBiomes(17)
river = forBiomes(7)

#Add desert to all types so other stuff doesn't appear as often
desert.addToGeneration("WARM", 2000)
desert.addToGeneration("COOL", 2000)
desert.addToGeneration("DESERT", 2000)
desert.addToGeneration("ICY", 2000)

#control spawns
desert.removeAllSpawns("CREATURE")
desert.addSpawn("net.minecraft.entity.passive.EntityPig", "CREATURE", 60, 1, 3)
desert.addSpawn("net.minecraft.entity.passive.EntityChicken", "CREATURE", 100, 2, 2)

#repeat for desertHills, not as often tho
hills.addToGeneration("WARM", 200)
hills.addToGeneration("COOL", 200)
hills.addToGeneration("DESERT", 200)
hills.addToGeneration("ICY", 200)

hills.removeAllSpawns("CREATURE")
hills.addSpawn("net.minecraft.entity.passive.EntityHorse", "CREATURE", 10, 1, 1)

#river spawns
river.removeAllSpawns("CREATURE")
river.addSpawn("net.minecraft.entity.passive.EntityCow", "CREATURE", 60, 1, 2)
river.addSpawn("net.minecraft.entity.passive.EntitySheep", "CREATURE", 40, 2, 2)

Tweaker.setStage("PRE_INIT")
desert.set("reedsPerChunk", 20)
desert.set("clayPerChunk", 10)
hills.set("clayPerChunk", 10)
Tweaker.setStage("FINISHED_LOAD")

#Remove other biomes
all = forAllBiomes()
all.set("genWeight", 11)
all.set("isSpawnBiome", false)

#remove all water (keeps lakes tho)
all.registerGenBlockRep("minecraft:water", "minecraft:air")

#final weighting - hills about 1/10 the time
desert.set("isSpawnBiome", true)
desert.set("genWeight", 50000)
hills.set("isSpawnBiome", true)
hills.set("genWeight", 5000)

**Comment by superckl** [comment]
I'm going to require a full log thenthen (on pastebin or similar). The file I need should be called "fml-client-latest" or something similar.

**Comment by superckl** [comment]
I was unable to reproduce this issue on my side with the setup you have described. Both normal and BOP world types generated the operate terrain:
![2015-09-04_16 28 06](https://cloud.githubusercontent.com/assets/2230216/9694860/0e35a1f0-5322-11e5-9a28-4229dc26b65a.png)

Something else is wrong. I'm going to guess your config is formatted incorrectly somewhere, or it's simply not being read.

**Comment by Outfluence** [comment]
Thanks so much for your responses! The test is supposed to add desert (sand) to all biomes, but I am not seeing that. The world generates as if BiomeTweaker is not installed. I checked and it IS showing up as an installed mod.
Here is a link to that file - https://www.dropbox.com/s/mpuayvtu7kdnrqu/fml-client-latest.log?dl=0

**Comment by Outfluence** [comment]
I also added Natura mod but I don't think it made a difference

**Comment by superckl** [comment]
Umm, I just noticed what file you said you were running... Where the hell did you get it? The mod ID is "Biomes Tweaker" and the version number is missing the build...

You should only be downloading BiomeTweaker from here:

http://minecraft.curseforge.com/mc-mods/228895-biometweaker

**Comment by superckl** [comment]
Are you using this, and thinking it's BiomeTweaker...?

http://minecraft.curseforge.com/mc-mods/224502-biomes-tweaker

**Comment by Outfluence** [comment]
I don't even know.... *crying face*

Pretty sure I got it from the first link.

**Comment by Outfluence** [comment]
Wait do you mean the testing file?

**Comment by superckl** [comment]
The jar you are using. It's not from me. **It's not even BiomeTweaker**. It's a completely different mod.

Download the latest version from here:

http://minecraft.curseforge.com/mc-mods/228895-biometweaker/files

Use that. Move your config to the correct folder. ???. Profit.

**Comment by Outfluence** [comment]
So listen up. In my mods folder I have a BiomeTweaker folder with configs inside that. Also in my mods folder I have a BiomesTweaker config file - WITH AN S. I deleted that file because its not from you. Then I started up minecraft and it came back, So I deleted every biometweaker mod from my mods folder and reopened minecraft AND IT CAME BACK!

Here is the file -
https://www.dropbox.com/s/5i9ea69qg4nq9ur/BiomesTweaker.cfg?dl=0
